ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomUpDownWidget.js

```diff
@@ -25,8 +25,8 @@ const CustomUpDownField = (props) => {
         key={props.id}
         value={props?.value}
         variant="standard"
-        onChange={e => props?.onChange(e.target.value)}
-        type="number"
+        onChange={e => e.target.value<0?props?.onChange(e.target.value=""):props?.onChange(e.target.value)}
+        type="tel"
```

Comment:
Curious: what is a type "tel"? Telephone number?

Reply:
The reason to remove the "number" was to remove the painpoints that user experience with the input field with that updown  arrow , which probably noone cares to click .

![image](https://user-images.githubusercontent.com/71957674/191993214-83335f5f-736c-48df-a7b4-c8c9263936d4.png)

Reply:
Use of the up/down arrows is option, right? Users have the choice of up/down arrows or entering a number. It's good to have this choice.

Reply:
Thank you for this screenshot. It's helpful.

Reply:
We don't want the user to click the down arrow, but removing the arrow doesn't prevent them from entering a negative number either.

Changing the field type is not a good solution and likely to cause issues - "tel" is generally, not a type that we'll be concerned with for a couple years.